slips_files/common/parsers/config_parser.py

```diff
@@ -469,7 +472,7 @@ def local_ti_data_path(self):
         return self.read_configuration(
```

Comment:
the read_configuration function is not modified by youright? this function should be modified to read from the parsed yaml because now its expecting to read from the old config files